src/modules/app/scanner.js

```diff
@@ -4,7 +4,7 @@ import omit from 'lodash/omit';
 
```

Comment:
I have now refactored it to be a higher order component. So now I think its more a question if the first letter should upper case or not since the default export is a function that creates a component.